subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -1009,91 +1013,63 @@ static void isr_done_cleanup(void *param)
 	ticker_stop(TICKER_INSTANCE_ID_CTLR, TICKER_USER_ID_LLL,
 		    TICKER_ID_SCAN_STOP, NULL, NULL);
 
-#if defined(CONFIG_BT_CTLR_ADV_EXT)
-	struct event_done_extra *extra;
-
-	if (lll->is_aux_sched) {
-		struct node_rx_pdu *node_rx;
-
-		node_rx = ull_pdu_rx_alloc();
-		LL_ASSERT(node_rx);
+#if defined(CONFIG_BT_CTLR_SCAN_INDICATION)
+	struct node_rx_hdr *node_rx = ull_pdu_rx_alloc_peek(3);
```

Comment:
Can you please add a comment on why we need 3 nodes here? That has been very useful the other times where you've commented on that :)

Reply:
Added comment.